### DISCUSSION TITLE
WIN 11 24H2

### DISCUSSION THREAD

**Discussion by rapidfire-zyrex** [Atlas-OS/Atlas#1242] [Help & Support]
I HAVE JUST DOWNLOADED **WIN 11 24H2** ,HOW TO USE PLAYBOOK ON IT. WIN 11 24H2 IS NOT SUPPORTING THE PLAYBOOK.

**Comment by pencilnav** [comment]
Be patient. We are working on a solution.

Also duplicate of https://github.com/orgs/Atlas-OS/discussions/1238
